### PR TITLE
chore(monorepo): make webpack output bundles

### PR DESCRIPTION
**Pull request by steveoh**
just messing around with webpack

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (stdavis) [inline .babelrc:1]: Isn't this a duplicate of `babel.config.js`?

File: packages/layer-selector/src/index.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import LayerSelector from './LayerSelector';
```
Comment (stdavis) [inline packages/layer-selector/src/index.js:1]: This file seems a bit silly. Maybe webpack requires it?

Comment (steveoh) [reply packages/layer-selector/src/index.js:1]: yes webpack requires it

File: .babelrc (hunk 1 above)
Comment (steveoh) [reply .babelrc:1]: partially? I'm not sure if it is required or not by webpack

**Comment by steveoh** [comment]
So I don't know if this is any better or different from the babel output. I thought it was worth trying. What do you think about it?

**Comment by stdavis** [comment]
The output looks pretty much the same to me. Does it work better with jest in other projects?

**Comment by steveoh** [comment]
From what I can tell it does not fix any problems. the output looked different to me but I didn't look very close.

**Pull request closed without merging**